README.md

```diff
@@ -0,0 +1,50 @@
+[![Build Status][ci-img]][ci] [![Released Version][maven-img]][maven]
+
+# OpenTracing Apache HttpClient 
+
+Apache HttpClient instrumentation.
+
+## Configuration
+
+```java
+GlogalTracer.register(tracer);
+HttpClient httpClient = new TracingHttpClientBuilder()
```

Comment:
Wouldn't it be possible to use a `HttpRequestInterceptor`, instead of requiring consumers to use this special builder?